rateLimits/messageRateLimits.md

```diff
@@ -68,6 +74,34 @@ Content-Type: application/json
 }
 ```
 
+### Sample Delay Queue full Response
+
+You will receive this error when your message queue is full
```

Comment:
Need a colon for consistency here

"You will receive this error when your message queue is full:"